model/roles.go

```diff
@@ -338,18 +340,28 @@ func LoadRoleManifest(manifestFilePath string, releases []*Release) (*RoleManife
 				continue
 			}
 
-			role.Jobs = append(role.Jobs, job)
+			roleJob.Job = job
+
+			for name, info := range roleJob.Consumes {
+				info.Name = name
+				roleJob.Consumes[name] = info
```

Comment:
Isn't this already set? Why else would the `range` operator return this combination of `name` and `info`?

Reply:
Because this is loaded from user-provided role-manifest, where it's probably
```yaml
…
consumes:
    thing:
        # No `name` subkey here
        type: mysql
```

Reply:
Ah, I see now; it is the whole `range` returning values tripping me up again. ☹️ 

So to avoid copying structs back and forth it could be

```
for name := range roleJob.Consumes {
    roleJob.Consumes[name].Name = name
}
```

Reply:
Yes, except that as far as I can tell that's not valid in golang for reasons I don't understand. It just throws compiler errors at me.